src/main/java/com/suse/salt/netapi/calls/runner/Manage.java

```diff
@@ -27,6 +27,15 @@ private Manage() { }
                 new TypeToken<List<String>>(){});
     }
 
+    public static RunnerCall<List<String>> up(Optional<Integer> timeout,
+            Optional<Integer> gatherJobTimeout) {
```

Comment:
No, their semantics are the same - but in this case we cannot introduce this in a general way for all RunnerCalls, simply because those two parameters have been added "ad hoc" for [manage.up](https://github.com/saltstack/salt/pull/40018/files#diff-7ab664360b664be453b91fe52517b5a3R137) and [manage.status](https://github.com/saltstack/salt/pull/40018/files#diff-7ab664360b664be453b91fe52517b5a3R33).

@lucidd: is it OK to add them as Optionals or would method overloading make more sense here to be backwards compatible?